esp-hal/src/uart.rs

```diff
@@ -277,16 +277,14 @@ pub enum Error {
     ///
     /// This error occurs when the received data does not conform to the
     /// expected UART frame format.
-    #[allow(clippy::enum_variant_names, reason = "Frame error is a common term")]
-    RxFrameError,
+    RxFrame,
 
     /// A parity error was detected on the RX line.
     ///
     /// This error occurs when the parity bit in the received data does not
     /// match the expected parity configuration.
     /// with the `async` feature.
```

Comment:
unrelated and just docs but this half-sentence looks out of place and I also have to guess what it tries to tell me 🤔

Reply:
we don't even have an async-feature anymore